compiler/src/main/java/org/qbicc/context/CompilationContext.java

```diff
@@ -80,6 +80,25 @@ default ClassContext getClassContextForLoader(VmClassLoader classLoaderObject) {
      */
     void registerEntryPoint(ExecutableElement method);
 
+    /**
+     * InvokableElements are executable elements methods that must
+     * be included in the final image because they have either been
+     * entered into a dispatching table and thus may be invoked via
+     * an indirect call through a function pointer.
+     * These elements are not necessarily EntryPoints, but the program
+     * points that invoke them have been lowered to the point where it
+     * is no longer possible to determine they are reachable from the graph.
+     *
+     * @param element the element to register as invokable.
+     */
+    void registerInvokableElement(ExecutableElement element);
```

Comment:
I would maybe go with something more general here, e.g. `registerAutoQueuedElement` whose description is something like "Automatically enqueue the given element for all remaining phases", with an admonition "The element can never be eliminated so use sparingly" or something like that.